test/renderer/crud-store.spec.js

```diff
@@ -451,6 +451,7 @@ describe('store', () => {
 
```

Comment:
Would be probably good to have few tests only about the generation of the update query.